src/lib/trace/debug.h

```diff
@@ -17,14 +19,20 @@
 
 /* Send every event to a debug log level. This is useful to debug new trace
  * events without implementing them for a specific event tracing framework.
- * Note that the arguments are ignored since at this step we do not know the
- * types and amount there is. */
+ *
+ * NOTE: arguments can't be used becaue there is no easy generic ways to learn
```

Comment:
Could we take a format string and a bunch of args, and pass them into log_debug?   We'll have to concatenate the format string to the string we're passing in now.

Reply:
Hmmm so the tracepoint the way I made the API is:

`tor_trace(<subsystem_name>, <event_name>, args1, arg2, ...)`. The args1 is the information we want to record. So if we put `fmt` as args1, then all tracers will have to record `fmt`.